manage.py

```diff
@@ -16,5 +16,6 @@ def main():
     execute_from_command_line(sys.argv)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "extlinks.settings.production")
```

Comment:
Why are we setting a default here, and why are we defaulting to production?

Reply:
In trying to figure out why the upgrade wasn't working, I noticed that the `DJANGO_SETTINGS_MODULE` was defined in the .env file but it was never used in the application. I went to the Wikipedia Library code for reference and found it there (as well as in other StackOverflow solutions while researching the problems we were having).

Reply:
`DJANGO_SETTINGS_MODULE` is predefined environment variable handled by django core. Since we use docker-compose to load the `.env` file in all of our environments, there shouldn't be any execution environment that doesn't have this set already. My guess is that it's in `TWLight` from the old manual deployment days and that it could be remove. I believe this change isn't doing anything for us here.

Reply:
Got it! I've removed that line of code.